cmake/CMakeLists.txt

```diff
@@ -115,8 +114,8 @@ if(NOT YARP_NO_DEPRECATED) # since YARP 2.3.68
   endif()
 endif()
 
-# Remove renamed files from installation (since YARP 2.3.68)
-foreach(_f FindACE.cmake
+# Remove renamed files from installation
+foreach(_f FindACE.cmake                                          # since YARP 2.3.68
```

Comment:
I never noticed this "Remove renamed files from installation", but it is super useful, and it would be worth refactoring it in a YCM Module (great job for a CMake starter!).